rlschool/A1_robot/README.md

```diff
@@ -0,0 +1,95 @@
+# Quadrupedal
+Quadrupedal is a physical simulation platform for unitree A1 robot based on pybullet and https://github.com/google-research/motion_imitation.
+
+Please consider to cite this environment if it can help your research.
+
+<img src="demo.gif" width="400"/>
+
+## Install
+
+```python
+pip install rlschool
+```
+
+#### For local installation, execute following commands:
+
+```python
+git clone https://github.com/PaddlePaddle/RLSchool
+cd RLSchool
+pip install .
+```
+## Quick Start
+Quadrupedal environment follows the standard gym APIs to create, run and close an environment.
+
+```python
+# We show a simple example to start Quadrupedal here
+from rlschool import make_env
+import numpy as np
+env = make_env('Quadrupedal',render=1,task="stairstair")
+observation,info = env.reset()
```

Comment:
The original gym API does return `info`.

Reply:
?